enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/WildFlySwarmHealthCheckEnricher.java

```diff
@@ -0,0 +1,48 @@
+package io.fabric8.maven.enricher.fabric8;
+
+import io.fabric8.kubernetes.api.model.Probe;
+import io.fabric8.kubernetes.api.model.ProbeBuilder;
+import io.fabric8.maven.enricher.api.AbstractHealthCheckEnricher;
+import io.fabric8.maven.enricher.api.EnricherContext;
+
+import static io.fabric8.maven.core.util.MavenUtil.hasDependency;
+
+/**
+ * Enriches wildfly-swarm containers with health checks if the monitoring fraction is present.
+ */
+public class WildFlySwarmHealthCheckEnricher extends AbstractHealthCheckEnricher {
+
+    private static final int DEFAULT_HEALTH_PORT = 8080;
```

Comment:
You can define configurable properties as in https://github.com/fabric8io/fabric8-maven-plugin/blob/master/enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/DependencyEnricher.java#L62-L76 & then retrieve them as in https://github.com/fabric8io/fabric8-maven-plugin/blob/master/enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/DependencyEnricher.java#L172